package.json

```diff
@@ -11,21 +11,19 @@
     "@react-native-community/datetimepicker": "^3.0.9",
     "@types/react": "^16.14.0",
     "@types/react-native": "^0.63.35",
-    "mobx": "^4.13.0",
+    "mobx": "^6.7.0",
     "mobx-react": "^5.4.3",
-    "moment": "^2.24.0",
     "react": "^16.13.1",
-    "react-native": "^0.63.3",
-    "react-native-image-picker": "^3.1.3",
+    "react-native": "^0.63.5",
     "react-native-image-crop-picker": "^0.37.3",
     "react-native-keyboard-aware-scroll-view": "github:APSL/react-native-keyboard-aware-scroll-view#v0.9.2",
     "react-native-modal-datetime-picker": "^8.7.1",
     "react-native-webview": "^10.3.3",
     "typescript": "3.2.2"
   },
   "peerDependencies": {
-    "mobx": "*",
-    "mobx-react": "*",
+    "mobx": "^6.0.0",
+    "mobx-react": "^5.0.0",
```

Comment:
This looks to be incorrect based on `mobx` going to v6 
- https://www.npmjs.com/package/mobx-react